tox.ini

```diff
@@ -38,3 +42,6 @@ commands = flake8 raiden
 [flake8]
 max-line-length = 99
 max-complexity = 10
+
+[pep8]
+max-line-length = 99
```

Comment:
Hummm, if you don't mind these 3 extra lines I'm going to leave them there, pep8 is a requirement for the flake8 tool, that means both binaries are available if flake8 is installed and tools like syntastic will run both automatically, this makes life easier by having both tools consistent. Note: this happen even if pep8 is not installed directly and we are using just the pycodestyle package.